packages/react/package.json

```diff
@@ -27,7 +27,9 @@
     "./package.json": "./package.json",
     "./jsx-runtime": "./jsx-runtime.js",
     "./jsx-dev-runtime": "./jsx-dev-runtime.js",
-    "./": "./"
+    "./umd/*": "./umd/*",
+    "./unstable-shared-subset": "./unstable-shared-subset.js"
```

Comment:
Instead of exposing this, can we instead just use the deep linking into `/src/` for the bundle? Because that way it's not exposed upon publishing.

Also, deleting all `./src/*` entries after bundling is easier than any arbitrary name.

Reply:
Yeah that sounds good

Reply:
I'm going to open a separate PR for this so I can get it reviewed